drive/types.bal

```diff
@@ -475,18 +475,18 @@ type CreateFileOptional record {
     boolean useContentAsIndexableText?; 
 };
 
-# Update file optional parameters
+# Update file optional parameters.
 #
-# + ocrLanguage - A language hint for OCR processing during image import (ISO 639-1 code).
-# + removeParents - A comma-separated list of parent IDs to remove. 
+# + ocrLanguage - A language hint for OCR processing during image import (ISO 639-1 code)
+# + removeParents - A comma-separated list of parent IDs to remove
 # + keepRevisionForever - Whether to set the 'keepForever' field in the new head revision. 
 #                         This is only applicable to files with binary content in Google Drive. 
 #                         Only 200 revisions for the file can be kept forever. If the limit is reached,
 #                         try deleting pinned revisions. (Default: false)
 # + useContentAsIndexableText - Whether to use the uploaded content as indexable text. (Default: false)
 # + includePermissionsForView - Specifies which additional view's permissions to include in the response. 
-#                               Only 'published' is supported.
-# + addParents - A comma-separated list of parent IDs to add.
+#                               Only 'published' is supported
+# + addParents - A comma-separated list of parent IDs to add
 # + supportsAllDrives -  Whether the requesting application supports both My Drives and shared drives. (Default: false)  
```

Comment:
Replace "Whether" with "Indicates whether".

Reply:
Hi,
We have used the same content from [1] for parameter descriptions
[1]https://developers.google.com/drive/api/v3/reference/files
Thanks